### PR TITLE
Deb build scripts added + deb guide

### LINKED ISSUE #1519: Ubuntu package
We should build ubuntu packages for ralph (without much of scan plugins) to be able to install easily if you're reluctant to use docker.
- all js and components integrated into the package
- /etc/ralph for system configuration
- only ubuntu supported

### PR DESCRIPTION
**Pull request by vi4m**
Fixes #1519
- ability to install ralph with `apt-get install ralph-core`
- Buildcripts for issuing new ralph package from within vagrant env.
- Documentation - not only deb installation, but improved structure, merged some pages, and refactor
- removed fix-tablib from Makefile - put into the requirements
- setup.py no longer uses requires, you should install from requirements.txt by hand
- moved all docker/configuration text into single instllation file
- subdir development/ for dev pages
- basic markdown extensons config added
- added `prod` settings handling

### CODE REVIEW

**Comment by vi4m** [comment]
Please review it, and i will squash it

File: docs/development/packaging.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Issuing the new deb snapshot
+
+Inside the Vagrant:
+
+1. Make sure you have bintray api key set - `export BINTRAY_APIKEY=APIKEYHERE` for binary upload.
+2. `make package` builds package ./packaging/build* and then upload it ./packaging/upload* to the bintray.
+3. Build name will be something like "ralph-3.0.0-#BUILDNUMBER"
+
+Note: Internally we run `dch` debian tool to help us generate build number, and generate debian/changelog. file.
+After issuing a new 'major' release it is required to manually pre-set new version in this file.
```
Comment (ghost) [inline docs/development/packaging.md]: ".. in this file" - what file are you referencing?

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Source: ralph
+Section: python
+Priority: extra
+Maintainer: pylabs@allegrogroup.com
+Build-Depends: debhelper (>= 9), python
+Standards-Version: 3.9.5
+
+Package: ralph-core
+Architecture: amd64
+Pre-Depends: dpkg (>= 1.16.1), python2.7 | python2.6, ${misc:Pre-Depends}
```
Comment (ghost) [inline debian/control]: why python2?

File: docs/development/packaging.md (hunk 1 above)
Comment (vi4m) [inline docs/development/packaging.md]: debian/changelog - this file contains the version number(weird, i know, but debian does it)

**Comment by mkurek** [comment]
Do you have to introduce 2 new top-level directories (debian and packaging)? Could you merge them into one directory?

**Comment by vi4m** [comment]
No. debian have to sit in the main sources directory.

**Comment by ghost** [comment]
:ok:

**Comment by vi4m** [comment]
ping

**Comment by ar4s** [comment]
:ok:

**Pull request merged**